api/controllers/MerchStoreController.ts

```diff
@@ -190,16 +190,18 @@ export class MerchStoreController {
     return { error: null, order };
   }
 
+  @Get('/orders/all')
+  async getAllMerchOrders(@AuthenticatedUser() user: UserModel): Promise<GetMerchOrdersResponse> {
+    if (!(PermissionsService.canAccessMerchStore(user)
+      && PermissionsService.canSeeAllMerchOrders(user))) throw new ForbiddenError();
+    const orders: OrderModel[] = await this.merchStoreService.getAllOrdersForAllUsers();
+    return { error: null, orders: orders.map((o) => o.getPublicOrder()) };
```

Comment:
nit: we don't need the explicit typing of `const orders: OrderModel[]` here, since we know the return type of `getAllOrdersForAllUsers()` is `OrderModel[]`